### DISCUSSION TITLE
Copilot remove closing parentheses

### DISCUSSION THREAD

**Discussion by Abdelraman** [community/community#130513] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

after working with copilot for a couple of months, I found some annoying issue regarding the closing parentheses
for the example the below code

dd()

then entering inside dd parentheses

dd() it just request->all() -->which is great

on pressing tab 

it should print dd(request()->all())

but it prints dd(request->all()    --> ) a closing parentheses is missing it was added by the IDE

using phpstorm

**Comment by Anamicca23** [comment]
m also encountering same issue this can be becasue of PHPStorm Settings! Head to Preferences or Settings, then go to Editor, General, and Smart Keys. Make sure "Insert pair bracket" and similar settings are correct...maybe
or maybe try ..to ..disable Smart Insertion of Pair Brackets! In Preferences or Settings under Editor, General, and Smart Keys, then go for unchecking automatic closing parentheses..this colud be done or uhh can try  to adjust Copilot or Plugin Settings! Check Copilot or plugin settings for code completion issues. Look for updates that might fix this . bcz i try all above and lastly  m looking forword that some one can fix it or will do manually in ide.

**Reply by Abdelraman** [reply]
sorry what do you mean by manually (do you mean typing it by yourself)
I tried all the options you mentioned but nothing changed, it even appear in if statements 
if (request->isAjax() --> missing the closing parentheses

**Comment by Abdelraman** [comment]
Any updates for that ?

**Comment by tpope** [comment, marked as answer]
This should be fixed in version 1.5.15.6094 of the JetBrains extension.
